source/administration/command-line-tools.rst

```diff
@@ -989,6 +992,29 @@ mattermost import slack
     .. code-block:: none
 
       ./mattermost import slack myteam slack_export.zip
+      
+mattermost integrity
+--------------------
+
+  Description
+    Command to check database schema integrity as well as referential integrity of channels, command web hooks, posts, schemes, sessions, users, and teams. This process may temporarily affect live system performance, and should be used during off-peak periods. 
+    
+  Format
+    .. code-block:: none
+
+      mattermost integrity {flag} {flag}
+            
+  Example
+    .. code-block:: none
+
+      ./mattermost integrity --confirm --verbose 
+      
+  Options
+    .. code-block:: none
+
+          --confirm   Confirm you really want to run a complete integrity check that may temporarily harm system performance.
```

Comment:
Are either of these optional?

Reply:
> Are either of these optional?

That's a good question. They are not required to be used simultaneously (you can run --confirm and then --verbose), but in terms of generating information I'd say that --confirm is optional as the check can proceed without it.

Reply:
Makes sense, if either (or both) are optional, we can label them as such. You can find an example of how to mark flags as optional here: https://docs.mattermost.com/administration/command-line-tools.html#mattermost-sampledata

PS: I'm wondering if `--confirm` is required -- if it's optional when would an admin want to use the flag? Would it be for automated scripts?

Reply:
That wasn't explicitly stated, so I have asked for confirmation. It would make sense that that would be the use case for it though.